src/containers/ImageUploader/components/ImageContent.tsx

```diff
@@ -6,19 +6,70 @@
  *
  */
 
+import { FieldHelperProps, useFormikContext } from "formik";
 import { useTranslation } from "react-i18next";
-import { FieldLabel, FieldRoot, FieldErrorMessage, FieldTextArea } from "@ndla/primitives";
+import { FileListLine } from "@ndla/icons";
+import { Button, FieldLabel, FieldRoot, FieldErrorMessage, FieldTextArea } from "@ndla/primitives";
+import { HStack } from "@ndla/styled-system/jsx";
 import { ImageUploadFormElement } from "./ImageUploadFormElement";
 import { FormField } from "../../../components/FormField";
 import { FormContent } from "../../../components/FormikForm";
+import { AI_ACCESS_SCOPE } from "../../../constants";
+import { useSession } from "../../../containers/Session/SessionProvider";
+import { useGenerateAltTextMutation } from "../../../modules/llm/llmMutations";
 import { TitleField } from "../../FormikForm";
+import { ImageFormikType } from "../imageTransformers";
 
 interface Props {
   language: string;
 }
 
 const ImageContent = ({ language }: Props) => {
   const { t } = useTranslation();
+  const { userPermissions } = useSession();
+  const { values, setStatus } = useFormikContext<ImageFormikType>();
+  const generateAlttextMutation = useGenerateAltTextMutation();
+
+  const generateAltText = async (helpers: FieldHelperProps<string | undefined>) => {
+    if (!values.imageFile) {
+      helpers.setError(t("textGeneration.errorImage"));
+      return null;
+    }
+
+    let image;
+    try {
+      if (typeof values.imageFile === "string") {
+        // We use the timestamp to avoid caching of the `imageFile` url in the browser
+        const timestamp = new Date().getTime();
+
+        const result = await fetch(values.filepath || `${values.imageFile}?width=2000&ts=${timestamp}`);
```

Comment:
Skal vi bare ha timestamp på en av de to mulighetene?

Reply:
Ja, for det er kun imageFile som potensielt kan være mega svær siden det er et bildet som en streng. Filepath skal være fint, dette er bare imageSrc fra `ImageUploadFormElement`